### ISSUE TITLE
Impute: impute with fixed value for all features

### ISSUE THREAD

**Issue by ajdapretnar** [biolab/orange3#5065]
- [ ] **What's your use case?**

I want to impute all missing values to 0. I don't want to do it feature by feature.

- [ ] **What's your proposed solution?**

Add an option to impute a specified value for all features. Custom-defined option is already available at single feature level, just add it to all features at once.

- [ ] **Are there any alternative solutions?**
Python Script.

**Comment by janezd** [comment]
Two designs (ignore the misalignments in the second). If the uses chooses this option, categorical values would be left missing (unless imputed below).

I guess I prefer the first, because it's more compact. Empty space in the second is ugly.

<img width="603" alt="Screenshot 2020-11-13 at 12 20 36" src="https://user-images.githubusercontent.com/2387315/99068290-86bbf680-25ac-11eb-9752-be75c8aca666.png">

<img width="603" alt="Screenshot 2020-11-13 at 12 33 51" src="https://user-images.githubusercontent.com/2387315/99068295-891e5080-25ac-11eb-9b70-3561bab1fae5.png">

**Comment by irgolic** [comment]
Go with the first one. I'd change the wording to 'Fixed values' (maybe also in individual attribute settings). Also, I'd put the three labels/lineedits into their own box.

The lineedits should be disabled if Values is not chosen. Although, perhaps it would look better if the box was hidden when Values is not chosen?

**Comment by janezd** [comment]
Wording: yes, Fixed values is better.

Disabling line edits: it's an option. But I also like UXs in which you can start editing the line edit and the corresponding line edit is automatically selected. As for hiding, we used to avoid it when it changes the total size of the widget.

I didn't understand the idea about boxes. Any of those? I think there are just too many boxes then.

<img width="603" alt="Screenshot 2020-11-13 at 12 59 30" src="https://user-images.githubusercontent.com/2387315/99070546-bb31b180-25b0-11eb-8242-d7ea6ce7993e.png">

<img width="603" alt="Screenshot 2020-11-13 at 12 59 30" src="https://user-images.githubusercontent.com/2387315/99070564-c4bb1980-25b0-11eb-87b9-d8fbc2fda44e.png">

**Comment by irgolic** [comment]
One box with three Labels/LineEdits in it.

I see your point about hiding, agreed. Regarding disabling, 'Fixed values' being automatically selected when editing a LineEdit is an option, I'm just worried it'll look disjoint (it won't be visually obvious that the LineEdits are related to the Fixed values RadioButton).

**Comment by janezd** [comment]
This widget currently does not impute string values. Should it? Does anybody consider this useful? (Hint: I hope for a "no".)

**Comment by ajdapretnar** [comment]
No.

You are welcome.

**Comment by janezd** [comment]
I owe you one at your next PR.

**Comment by ajdapretnar** [comment]
@janezd You are probably going to take those kind words from above back, but this fix doesn't seem to have any effect.

I tried it on three different data sets and nothing is ever imputed. :(
If it helps, my log says:
```
edit: index was invalid
edit: editing failed
```

Also, I wasn't obviously paying enough attention. Why have we decided to not add discrete variable? If I have 0 and 1, Orange would read those as discrete. So if I wanted to say "treat all missing as 'doesn't apply'", I am stuck. Even worse, fixed value for individual attribute only offers imputation with existing labels. What if I want to add 0? Perhaps I am troubling you too much. I can always do it with pandas, but perhaps this is useful enough to be in Orange. :)
